src/v/utils/log_hist.h

```diff
@@ -41,11 +41,32 @@
  * [32, 64)  = 2
  * [64, 128) = 0
  */
+class log_hist_base {
+public:
+    log_hist_base(int number_of_buckets, uint64_t first_bucket_upper_bound);
+
+    struct logform_config {
+        int64_t scale;
+        uint64_t first_bucket_bound;
+        int number_of_buckets;
+    };
+
+    seastar::metrics::histogram seastar_histogram_logform(logform_config) const;
+    seastar::metrics::histogram public_histogram_logform() const;
+    seastar::metrics::histogram internal_histogram_logform() const;
```

Comment:
Some comments here and above would be good explaining what "public" and "internal" refers to.

I assume it's for `/metrics` vs `/public_metrics`

Reply:
Good point, will add some method comments. You're right about it being `/metrics` vs `/public_metrics`